x-pack/plugins/security_solution/common/siem_migrations/model/api/rules/rule_migration.schema.yaml

```
@@ -34,7 +34,7 @@ paths:
                 properties:
                   migration_id:
                     description: The migration id created.
-                    $ref: '../../common.schema.yaml#/components/schemas/NonEmptyString'
+                    $ref: '../.././../../../common/api/model/primitives.schema.yaml#/components/schemas/NonEmptyString'
```

It looks like there's an unnecessary `/./` fragment to all the paths:
```suggestion
                    $ref: '../../../../../common/api/model/primitives.schema.yaml#/components/schemas/NonEmptyString'
```

---

yup, I probably accidentally added and copied that path everywhere 😅